react-charts/area-chart/step4.md

```diff
@@ -1,61 +1,34 @@
-Now we'll add multiple datasets to the area chart...
+Now you will add tooltips to the area chart.

```

Comment:
Let's instead write, "Add tooltips to the area chart." Let's follow that same sentence structure for the other instances of "Now you will" or "Next, you will do..." I don't want to add clutter with repeat comments, so just leaving the general note here :)